api/lib/domain/models/organizations-administration/OrganizationForAdmin.js

```diff
@@ -62,7 +65,9 @@ class OrganizationForAdmin {
     this.enableMultipleSendingAssessment = enableMultipleSendingAssessment;
     this.tags = tags;
     this.features = features;
-
+    if (this.type === 'SCO' && this.isManagingStudents) {
+      this.features[apps.ORGANIZATION_FEATURE.COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY.key] = true;
```

Comment:
```suggestion
      this.features[ORGANIZATION_FEATURE.COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY.key] = true;
```

également à la ligne 107